report_templates/registered_users.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+name: Registered users
+description: Report of registered users
+snippet: false
+template_inputs:
+- name: Users filter
+  required: false
+  input_type: user
+  description: Limit the report only on users found by this search query. Keep empty
+    for report on all available users.
+  advanced: false
+  value_type: search
+  resource_type: User
+model: ReportTemplate
+-%>
+<%- load_users(search: input('Users filter'), includes: :auth_source).each_record do |user| -%>
+<%-   report_row({
+         'Login': user.login,
```

Comment:
sorry for nitpicking, but the indentation should be by 2 spaces, this is 3 spaces comparing to the line above